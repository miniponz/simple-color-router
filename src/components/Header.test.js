import React from 'react';
import { shallow } from 'enzyme';
import Header from './Header';

describe('Header comp', () => {
  it(' renders Header', () => {
    const wrapper = shallow(<Header />);
    expect(wrapper).toMatchSnapshot();
  });
});
