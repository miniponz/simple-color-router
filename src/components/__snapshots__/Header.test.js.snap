// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header comp  renders Header 1`] = `
<header>
  <nav>
    <Link
      to="/blue"
    >
      Blue
    </Link>
    <Link
      to="/green"
    >
      Green
    </Link>
    <Link
      to="/purple"
    >
      Purple
    </Link>
  </nav>
</header>
`;
